src/Illuminate/Console/Scheduling/ScheduleRunCommand.php

```diff
@@ -60,11 +57,14 @@ public function __construct(Schedule $schedule)
     /**
      * Execute the console command.
      *
+     * @param  \Illuminate\Console\Scheduling\Schedule $schedule
```

Comment:
Add a space between the type and parameter name.

```suggestion
     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
```